#! /bin/sh /usr/share/dpatch/dpatch-run
## 025-setup.dpatch by Michal Čihař <nijel@debian.org>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: Adjust setup script to match Debian needs:
## DP:  - change config file path
## DP:  - do not check if config file dir is writable

@DPATCH@
diff -urNad phpmyadmin-3.1.0~/setup/lib/config_info.inc.php phpmyadmin-3.1.0/setup/lib/config_info.inc.php
--- phpmyadmin-3.1.0~/setup/lib/config_info.inc.php	2008-11-28 14:21:00.000000000 +0100
+++ phpmyadmin-3.1.0/setup/lib/config_info.inc.php	2008-11-30 12:48:58.000000000 +0100
@@ -15,7 +15,7 @@
 $cfg_db = array();
 
 // path to config file, relative to phpMyAdmin's root path
-$cfg_db['_config_file_path'] = './config/config.inc.php';
+$cfg_db['_config_file_path'] = '/var/lib/phpmyadmin/config.inc.php';
 
 /**
  * Value meaning:
@@ -138,4 +138,4 @@
     'InsertRows' => 'validate_positive_number',
     'ForeignKeyMaxLimit' => 'validate_positive_number',
     'Import/skip_queries' => 'validate_non_negative_number');
-?>
\ No newline at end of file
+?>
diff -urNad phpmyadmin-3.1.0~/setup/lib/index.lib.php phpmyadmin-3.1.0/setup/lib/index.lib.php
--- phpmyadmin-3.1.0~/setup/lib/index.lib.php	2008-11-28 14:21:00.000000000 +0100
+++ phpmyadmin-3.1.0/setup/lib/index.lib.php	2008-11-30 12:50:43.000000000 +0100
@@ -223,7 +223,7 @@
     $file_path = ConfigFile::getInstance()->getFilePath();
     $file_dir = dirname($file_path);
     $is_readable = true;
-    $is_writable = is_dir($file_dir) && is_writable($file_dir);
+    $is_writable = is_dir($file_dir);
     $file_exists = file_exists($file_path);
     if ($file_exists) {
         $is_readable = is_readable($file_path);
@@ -423,4 +423,4 @@
             PMA_lang('ZipDumpExportWarning', 'gzcompress'));
     }
 }
-?>
\ No newline at end of file
+?>
